docs/src/ci.md

```diff
@@ -0,0 +1,143 @@
+
+```
+CI_INTG_PKG_URL_<dep_name>: https://github.com/<user>/<dep_name>#<commit_hash>
+```
+
+You find the names of the environment variables in the section [Environment Variables](#Environment-Variables). For an example let's assume the name of the `dep` package is `dep1.jl`, `user1` forked the package and the commit hash of the fix for package `dep1.jl` is `45a723b`. Then, an example message could look like this:
+
+```
+This commit extends function foo with a new
+function argument.
+
+The function argument is required to control
+the new functionality.
+
+If you pass a 0, it has a special meaning.
+
+CI_INTG_PKG_URL_DEP1JL: https://github.com/user1/dep1.jl#45a723b
+```
+
+It is also possible to set a custom URL for more than one package, which depends on `orig`. Simple add an additional line with the shape of `CI_INTG_PKG_URL_<dep_name>: https://github.com/<user>/<dep_name>#<commit_hash>` to the commit message.
+
+!!! note
+
+    You don't need to add a new commit to set custom URLs. You can modify the commit message with `git commit --amend` and force push to the branch. This also starts the CI pipeline again.
+
+# Environment Variables
+
+The following table shows the names of the environment variables to use custom URLs for the unit and integration tests.
+
+Package Name    | Unit Test                      | Integration Test
+----------------|--------------------------------|-------------------------------
+QEDbase.jl      | `CI_UNIT_PKG_URL_QEDbase`      | `CI_INTG_PKG_URL_QEDbase`
+QEDevents.jl    | `CI_UNIT_PKG_URL_QEDevents`    | `CI_INTG_PKG_URL_QEDevents`
+QEDfields.jl    | `CI_UNIT_PKG_URL_QEDfields`    | `CI_INTG_PKG_URL_QEDfields`
+QEDprocesses.jl | `CI_UNIT_PKG_URL_QEDprocesses` | `CI_INTG_PKG_URL_QEDprocesses`
+
+# Unit Tests for CI Develops
+
+In this section, we explain how the unit tests are prepared and executed. It is not mandatory to read the section if you only want to use the CI.
+
+Before the unit tests are executed, the `SetupDevEnv.jl` is executed, which prepares the project environment for the unit test. It reads the `Project.toml` of the current project and adds the version of the `dev` branch of all QED dependency (`Pkg.develop()`) if no line starting with `CI_UNIT_PKG_URL_` was defined in the commit message. If `CI_UNIT_PKG_URL_` was defined, it use a custom URL.
```

Comment:
```suggestion
Before the unit tests are executed, the `SetupDevEnv.jl` is executed, which prepares the project environment for the unit test. It reads the `Project.toml` of the current project and adds the version of the `dev` branch of all QED dependency (`Pkg.develop()`) if no line starting with `CI_UNIT_PKG_URL_` was defined in the commit message. If `CI_UNIT_PKG_URL_` was defined, it will use the custom URL.
```